backend/src/core/features/game-identifier/game.identifier.spec.js

```diff
@@ -16,484 +16,206 @@ import { GamesAggregate } from "../../models/games.aggregate.js";
-          );
-          this.gamesRepository = createGamesRepositoryMock();
-          this.historyChecksRepository = createHistoryChecksRepositoryMock();
-
-          this.identifier = new GameIdentifier(
-            this.steamClient,
-            this.steamAppsRepository,
-            this.gamesRepository,
-            this.historyChecksRepository,
-            createLoggerMock(),
-            createConfigMock().features,
-            parseHTML,
-          );
-
-          await this.identifier.checkIfGameViaSource(this.source);
-        });
-
-        it("getSourceUntriedFilteredSteamApps was called once", function () {
-          expect(
-            this.steamAppsRepository.getSourceUntriedFilteredSteamApps,
-          ).toHaveBeenCalledTimes(1);
-        });
-
-        it("getSourceUntriedFilteredSteamApps was called with the correct arguments", function () {
-          expect(
-            this.steamAppsRepository.getSourceUntriedFilteredSteamApps,
-          ).toHaveBeenCalledWith(1, this.source);
-        });
-
-        it("getSourceHtmlDetailsPage was called twice", function () {
-          expect(this.steamClient.getSourceHtmlDetailsPage).toHaveBeenCalledTimes(2);
-        });
-
-        it("getSourceHtmlDetailsPage was called with the correct arguments", function () {
-          expect(this.steamClient.getSourceHtmlDetailsPage).toHaveBeenCalledWith(
-            this.steamApps.content[0].appid,
-            this.source,
-          );
-          expect(this.steamClient.getSourceHtmlDetailsPage).toHaveBeenCalledWith(
-            this.steamApps.content[1].appid,
-            this.source,
-          );
-        });
-
-        it("insertManyGames was called once", function () {
-          expect(this.gamesRepository.insertManyGames).toHaveBeenCalledTimes(1);
-        });
-
-        it("insertManyGames was called with the correct argument", function () {
-          expect(this.gamesRepository.insertManyGames).toHaveBeenCalledWith(this.games);
-        });
-
-        it("insertManyHistoryChecks was called once", function () {
-          expect(
-            this.historyChecksRepository.insertManyHistoryChecks,
-          ).toHaveBeenCalledTimes(1);
-        });
-
-        it("insertManyHistoryChecks was called with the correct argument", function () {
-          expect(
-            this.historyChecksRepository.insertManyHistoryChecks,
-          ).toHaveBeenCalledWith(this.historyChecks);
-        });
-
-        it("updateSteamAppsById was called once", function () {
-          expect(this.steamAppsRepository.updateSteamAppsById).toHaveBeenCalledTimes(1);
-        });
-
-        it("updateSteamAppsById was called with the correct argument", function () {
-          expect(this.steamAppsRepository.updateSteamAppsById).toHaveBeenCalledWith(
-            this.steamApps.content,
-          );
-        });
+  describe(".checkIfGameViaSteamWeb.", function () {
+    describe("Finds no unidentified steam apps in the database", function () {
```

Comment:
Ok, lets refactor the tests also properly. The tests are to detailed (from long time before :)). When this method returns after `steamApps.isEmpty` the only things we care about are:
- no call to steam client
- nothing is persisted

All other checks can be removed.